sonar-iac-plugin/src/main/resources/static/documentation.md

```diff
@@ -18,6 +18,23 @@ Discover and update the Terraform [properties](/analysis/analysis-parameters/) i
 * CloudFormation with AWSTemplateFormatVersion 2010-09-09 (YAML and JSON)
 * AWS, Azure and GCP
 
+### Terraform provider versions
+
+The respective Terraform providers are frequently updated. New resources, properties and default values are added. At the same time, others are deprecated or dropped. For this reason, the Terraform analysis is defensive by default.
+Some issues will be automatically silenced to avoid raising False Positives. In order to get a more precise analysis you can specify the provider versions your code supports via a parameter.
```

Comment:
```suggestion
The respective Terraform providers are frequently updated. New resources, properties and default values are added. At the same time, others are deprecated or dropped. For this reason, the Terraform analysis is defensive by default: some issues will be automatically silenced to avoid raising false positives. In order to get a more precise analysis you can specify the provider versions your code supports via a parameter.
```